src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
```

Comment:
In ur reset file can be more. 
For example: https://meyerweb.com/eric/tools/css/reset/
As u can see its css but u already know scss can easly read normal css :)
there u have thinks for example like: margin and padding resets for list that can be preety painfull to delete manualy when you want to use them